primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.commandlink.js

```diff
@@ -56,6 +57,7 @@ PrimeFaces.widget.CommandLink = PrimeFaces.widget.BaseWidget.extend({
                 }
                 if (PrimeFaces.ajax.Utils.isXhrSource($this, settings)) {
                     $this.jq.addClass('ui-state-loading');
+                    $this.ajaxStart = Date.now();
```

Comment:
Should this value be cleared somewhere once its no longer user so the value in memory is not misleading?